kano_feedback/kano_feedback_cli.py

```diff
@@ -15,13 +17,28 @@
 from kano_feedback.return_codes import RC
 
 
+def _check_can_create_file_flag(flag_path):
```

Comment:
Might be a better way than this (especially as it both allows the possibility of a half state and also essentially tests twice) but if will suffice for now.